test/response-schema.spec.ts

```diff
@@ -71,20 +71,16 @@ describe('response schema', () => {
       expect(response.statusCode).toBe(500);
       expect(response.json()).toMatchInlineSnapshot(`
         {
-          "details": {
-            "error": [
-              {
-                "code": "invalid_type",
-                "expected": "undefined",
-                "message": "Expected undefined, received object",
-                "path": [],
-                "received": "object",
-              },
-            ],
-            "method": "GET",
-            "url": "/incorrect",
-          },
           "error": "Internal Server Error",
+          "issues": [
```

Comment:
this is less useful than what we had before.

does the new error structure no longer expose the url and the method?

Reply:
Well we could also throw the old error here, but it is mostly for consistency. This type of error should also result in a 500 Internal server error, so I think it is not important to add the url and method to the error. The default error reporting from fastify reports these values anyways with the error.

Reply:
![image](https://github.com/user-attachments/assets/8ab50b94-f52b-4779-afd4-6b5b7cae6248)

Plus you can just get those values from the error handler.

Reply:
will need to check it by hand in greater detail to understand whether this is sufficient.

I am primarily concerned about getting good bugsnag entries and logs,  i n c l u d i n g on the client side.

whatever solution we end up with, it should expose no less data than the current one.